From d9c7dd5db1d0f6f212072625781dddbbe6e253d7 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 13 Apr 2017 16:46:36 +0100
Subject: [PATCH 33/37] staging: rtl8723bs: clean up identical code on an if
 statement

The two different paths for an if statement are identical and hence
we can just replace it with the single statement.

Detected by CoverityScan, CID#1428443 ("Identical code for
different branches")

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Reviewed-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/core/rtw_mlme.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

--- a/drivers/staging/rtl8723bs/core/rtw_mlme.c
+++ b/drivers/staging/rtl8723bs/core/rtw_mlme.c
@@ -1093,11 +1093,7 @@ void rtw_free_assoc_resources(struct ada
 		rtw_init_bcmc_stainfo(adapter);
 	}
 
-	if (lock_scanned_queue) {
-		find_network(adapter);
-	} else {
-		find_network(adapter);
-	}
+	find_network(adapter);
 
 	if (lock_scanned_queue)
 		adapter->securitypriv.key_mask = 0;
